examples/blog/api.ttl

```diff
@@ -81,3 +81,30 @@
 <comments> a hydra:Link;
   hydra:supportedOperation
       <comment#post>.
+
+<Category> a hydra:Class ;
+  hydra:supportedOperation
+    <category#get>;
+  hydra:supportedProperty [
+    hydra:property <post>
+  ] ;
+  hydra:supportedProperty [
+    hydra:property <pinned-post>
+  ]
+.
```

Comment:
I added Category class with those two properties to test the behavior of resources only used as objects of those properties